parsec/backend/asgi/__init__.py

```diff
@@ -123,7 +129,8 @@ async def serve_backend_with_asgi(
             # Timestamp is added by the log processor configured in `parsec.logging`,
             # here we configure peer address + req line + rep status + rep body size + time
             # (e.g. "GET 88.0.12.52:54160 /foo 1.1 404 823o 12343ms")
-            "access_log_format": "%(h)s %(r)s %(s)s %(b)so %(D)sus",
+            "logger_class": ParsecLogger,
+            "access_log_format": "%(h)s %(r)s %(s)s %(b)so %(D)sus (author: %(author)s)",
```

Comment:
would be better to avoid having `(author: anonymous or invalid)` in the log when the author is not available.

For this you can put the ` (author: %(author)s)` formatting logic within `ParsecLogger`, and only have `"access_log_format": "%(h)s %(r)s %(s)s %(b)so %(D)sus%(author)s",` here

Reply:
Done.